exercism/bob/bob.src.js

```diff
@@ -0,0 +1,16 @@
+var Bob = function() {};
+
+  Bob.prototype.hey = function(what) {
```

Comment:
if statements do not need to be followed by a semicolon. remove semicolons that follow if statements like this `if (x) {  };`